### PR TITLE
OCPBUGS-30802: ipsec: Update the ipsec tasks to ensure that they are idempotent

### PR DESCRIPTION
**Pull request by barbacbd**

### CODE REVIEW

File: images/installer/Dockerfile [hunk 1]
```diff
@@ -38,7 +38,7 @@ RUN ln -s /usr/bin/pip3.11 /usr/bin/pip3
 RUN pip3 install --no-cache-dir boto3 botocore ansible-core
 
 # ansible-galaxy will not be available until ansible-core is installed
-RUN ansible-galaxy collection install -p /usr/share/ansible/collections amazon.aws && \
+RUN ansible-galaxy collection install -p /usr/share/ansible/collections amazon.aws community.general && \
```
Comment (patrickdillon) [inline images/installer/Dockerfile:41]: we're going to drop these usages of `community.general` from this pr right as we are no longer using this module?

File: .travis.yml [hunk 2]
```diff
@@ -9,13 +9,15 @@ before_cache:
...
-  - "3.7"
+  - "3.9"
 
 before_install:
   - python -m pip install --upgrade virtualenv
 
 install:
   - pip install --upgrade pip
+  - pip install -r requirements.txt
+  - ansible-galaxy collection install community.general
```
Comment (patrickdillon) [inline .travis.yml:20]: we're going to drop these usages of `community.general` from this pr right as we are no longer using this module?

**Comment by sdodson** [comment]
https://github.com/openshift/ovn-kubernetes/pull/2083 seems to have removed the rhel8 binaries from OVN Kube CNI. We likely need to get that reverted.

**Comment by sdodson** [comment]
/test e2e-aws-workers-rhel8
Fixes have landed in an accepted 4.16 payload, but the periodic hasn't run yet to confirm success, but this should pass now.

**Review by MayXuQQ — APPROVED** [review]

**Pull request merged**